File: java_src/DH3DLibrary/src/jp/spec/darkhorse2/dh3d/base/RigidBody.java
/**
 * 
 */
package jp.spec.darkhorse2.dh3d.base;

/**
 * @author Norihiro
 *
 */
public class RigidBody {
	public String name;
	public int boneIndex;
	public int groupIndex;
	public int groupTarget;
	public int shapeType;
	public float shapeW;
	public float shapeH;
	public float shapeD;
	public DHVector3 position;
	public DHVector3 rotate;
	public float weight;
	public float positionDim;
	public float rotateDim;
	public float recoil;
	public float friction;
	public int type;
	
	public RigidBody(){
		
	}
}

File: src/js/base/RigidBody.js
/*--------------------------------------------------------------------------------
 * DH3DLibrary RigidBody.js v0.2.0
 * Copyright (c) 2010-2012 DarkHorse
 *
 * DH3DLibrary is freely distributable under the terms of an MIT-style license.
 * For details, see the DH3DLibrary web site: http://darkhorse2.0spec.jp/dh3d/
 *
 *------------------------------------------------------------------------------*/
var RigidBody = Class.create({
  // FIXME: field name
  name: '',
  boneIndex: 0,
  groupIndex: 0,
  groupTarget: 0,
  shapeType: 0,
  shapeW: 0.0,
  shapeH: 0.0,
  shapeD: 0.0,
  position: null,
  rotate: null,
  weight: 0.0,
  positionDim: 0.0,
  rotateDim: 0.0,
  recoil: 0.0,
  friction: 0.0,
  type: 0,

  initialize: function() {
  },
});

